lib/grape-swagger/endpoint.rb

```diff
@@ -198,7 +198,7 @@ def response_object(route)
         response_model = expose_params_from_model(value[:model]) if value[:model]
 
         if route.request_method == 'DELETE' && !value[:model].nil?
```

Comment:
habe this change (https://github.com/ruby-grape/grape-swagger/pull/583/files#diff-7af39528837333c960830618fb6a52e9L192) must be reverted